shell/gpu/gpu_rasterizer.h

```diff
@@ -40,11 +41,16 @@ class GPURasterizer : public Rasterizer {
   flow::CompositorContext compositor_context_;
   std::unique_ptr<flow::LayerTree> last_layer_tree_;
   ftl::WeakPtrFactory<GPURasterizer> weak_factory_;
```

Comment:
The weak factory must always be the last member variable. The reason is detailed in the [header for the the same here](https://fuchsia.googlesource.com/ftl/+/master/memory/weak_ptr.h#116).